tests/containers/podman_integration.pm

```diff
@@ -19,6 +19,28 @@ use containers::common;
 
 my $test_dir = "/var/tmp";
 my $podman_version = "";
+my $quadlet = "";
+
+sub run_tests {
+    my ($log_file, $skip_tests) = @_;
+
+    my $remote = ($log_file =~ /remote/) ? 1 : 0;
+    my $args = ($log_file =~ /root/) ? "--root" : "--rootless";
+    $args .= " --remote" if ($remote);
```

Comment:
Same argument as above. I would much rater have `$remote` as explicit parameter and then build the log filename based on them, than try to deduce information based on the filename.

Reply:
Looks good, thanks!

Reply:
Fixed.